contracts/ProtocolReserve/ProtocolShareReserve.sol

```diff
@@ -125,10 +132,13 @@ contract ProtocolShareReserve is AccessControlledV8, ReentrancyGuardUpgradeable,
     /**
      * @dev Initializes the deployer to owner.
      * @param _accessControlManager The address of ACM contract
+     * @param _loopsLimit Limit for the loops in the contract to avoid DOS
      */
-    function initialize(address _accessControlManager) external initializer {
+    function initialize(address _accessControlManager, uint256 _loopsLimit) external initializer {
+        if (_loopsLimit == 0) revert InvalidMaxLoopsLimit();
```

Comment:
I think this check is not needed. If `_loopsLimit` is 0 then `_setMaxLoopsLimit` will fail because there is a `require` statement comparing the new value with the current one, that initially will be zero